ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -101,6 +101,7 @@ def __init__(
     @classmethod
     def create(cls, config_dict: dict, metadata_config: OpenMetadataServerConfig):
         config = MetadataRestSinkConfig.parse_obj(config_dict)
+        metadata_config = OpenMetadataServerConfig.parse_obj(metadata_config)
```

Comment:
This is not needed, we are already passing the class `OpenMetadataServerConfig`, not the dict. As we can read everything typed from the JSON Schemas, we can go and reduce the parsings we needed in the beginning